src/components/modals/StatusIndicator.vue

```diff
@@ -0,0 +1,23 @@
+<template lang="html">
+  <div class="box w-25 bg-light drop-shadow border border-primary p-4 text-center">
+    {{ status }}
+    <i-spinner class="mt-4"></i-spinner>
+  </div>
+</template>
+
+<script>
+
+export default {
+  computed: {
+    status() {
+      return this.$store.state.processing_message;
+    },
+  },
+};
+</script>
+
+<style lang="scss">
+  .box {
```

Comment:
ehehe it is not centered... What a nice position absolute with left 50%, top 50% and negative margin left and margin top? :D